### DISCUSSION TITLE
Add a possibility to make sub-collections

### DISCUSSION THREAD

**Discussion by noisenlofi** [ppy/osu#21770] [Ideas]
Instead of having to make numerous of map collections, players should have to be able to make sub-collections. A Quality of Life addition idea.

Ex: Training Collection: 
         - Finger Control (sub-collection)
         - stream (sub-collection)
         - jump (sub-collection)

**Comment by soopax** [comment]
I think this is discussed in https://github.com/ppy/osu/discussions/15397.
